File: python/8 kyu/208.py
# https://www.codewars.com/kata/576b93db1129fcf2200001e6/train/python

# Passed

def sum_array(arr):
    result = sum(sorted(arr)[1:-1]) if arr and len(arr) > 2 else 0
    return result

output = sum_array([6, 2, 1, 8, 10])
print(output)

File: nodejs/7 kyu/208.js
// https://www.codewars.com/kata/56b8903933dbe5831e000c76/train/javascript

// Passed

function spoonerize(words) {
  let [firstWord, secondWord] = words.split(" ");
  let [modifiedFirstWord, modifiedSecondWord] = [
    secondWord[0] + firstWord.slice(1),
    firstWord[0] + secondWord.slice(1),
  ];
  let result = `${modifiedFirstWord} ${modifiedSecondWord}`;
  return result;
}

const output = spoonerize("nit picking");
console.log(output);